packages/chain-adapters/src/types.ts

```diff
@@ -173,6 +173,19 @@ export type BuildDepositTxInput<T extends ChainId> = Omit<BuildSendTxInput<T>, '
   memo: string
 }
 
+type BuildLPTxInput<T extends ChainId> = Omit<BuildSendTxInput<T>, 'to' | 'value'> & {
+  poolId: string
+  shareOutAmount: string
+}
+
+export type BuildLPAddTxInput<T extends ChainId> = BuildLPTxInput<T> & {
+  tokenInMaxs: [cosmossdk.CosmosSDKToken, cosmossdk.CosmosSDKToken]
```

Comment:
This name feels strange, but I'm assuming this is some osmo vernacular I'm not across 👌